external/svc-etcd.yaml

```diff
@@ -0,0 +1,490 @@
+
+
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    labels:
+      postgres.database.fits.cloud/partition-id: sample
+      postgres.database.fits.cloud/managed-by: postgreslet
+      postgres.database.fits.cloud/component: etcd
+    name: patroni-etcd
+    namespace: postgreslet-system
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    labels:
+      postgres.database.fits.cloud/partition-id: sample
+      postgres.database.fits.cloud/managed-by: postgreslet
+      postgres.database.fits.cloud/component: etcd
+    name: patroni-etcd
+    namespace: postgreslet-system
+  rules:
```

Comment:
Review of required resources necessary

Reply:
Turns out, almost none of it is neccessary :-)